packages/api/amplify_api_dart/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -12,6 +12,8 @@ import 'package:amplify_api_dart/src/graphql/web_socket/types/connectivity_platf
 import 'package:amplify_api_dart/src/graphql/web_socket/types/subscriptions_event.dart';
 import 'package:amplify_api_dart/src/graphql/web_socket/types/web_socket_types.dart';
 import 'package:amplify_core/amplify_core.dart' hide SubscriptionEvent;
+// ignore: implementation_imports
+import 'package:amplify_core/src/config/amplify_outputs/api_outputs.dart';
```

Comment:
Q: Needing this ignore seems odd, are we sure this is the best way to import/export this type?

Reply:
I think this is the way we do in the monorepo when we do not want to export files/types from one package but still want to use them in other packages.